src/main/java/io/kestra/plugin/gcp/cli/GCloudCLI.java

```diff
@@ -34,25 +34,28 @@
 @Plugin(
         examples = {
                 @Example(
-                        title = "Create a cluster then list them using a service account",
+                        title = "Create a cluster then list them using a service account.",
                         code = {
+                                "projectId: my-gcp-project",
                                 "serviceAccount: \"{{ secret('gcp-sa') }}\"",
                                 "commands:",
-                                "  - gcloud container clusters create simple-cluster",
+                                "  - gcloud container clusters create simple-cluster --region=europe-west3",
                                 "  - gcloud container clusters list"
                         }
                 ),
                 @Example(
-                        title = "Create a GCS bucket",
+                        title = "Create a GCS bucket.",
                         code = {
+                                "projectId: my-gcp-project",
                                 "serviceAccount: \"{{ secret('gcp-sa') }}\"",
                                 "commands:",
-                                "  - gsutil mb gs://my-bucket"
```

Comment:
The gsutil command fails with the following error:

```
2024-03-17 19:51:12.942Creating gs://smantri-kestra-20240317/...
2024-03-17 19:51:13.266You are attempting to access protected data with no configured
2024-03-17 19:51:13.269credentials. Please visit https://cloud.google.com/console#/project
2024-03-17 19:51:13.273and sign up for an account, and then run the "gcloud auth login"
2024-03-17 19:51:13.275command to configure gsutil to use these credentials.
2024-03-17 19:51:13.627Command failed with code 1
```
Hence, used gcloud command which runs successfully.